src/components/FVApps/FVAppsPresentation.js

```diff
@@ -24,28 +24,43 @@ function FVAppsPresentation({ sitesWithApps }) {
             The wealth of language data uploaded by Indigenous communities onto
             their FirstVoices language sites can also be accessed through mobile
             apps. The app pulls content directly from FirstVoices and is
-            compatible with most phones, tablets, and laptops.
+            compatible with most phones, tablets, and laptops. The new apps can
+            be installed directly from the web.
```

Comment:
This is repetitive. I think we could remove the last sentence of this paragraph.